### DISCUSSION TITLE
Should we have a booth at event X

### DISCUSSION THREAD

**Discussion by claredillon** [InnerSourceCommons/InnerSourceMarketing#243] [General]
From time to time the question comes up - should ISC have a booth at event X?

We had this discussion a few months ago in a marketing call, and I wanted to capture some of the thoughts I shared there. Please feel free to leave your own opinion here or insights that might help build the argument for or against...

There are a number of benefits of ISC having a booth at an event:
1. Brand awareness
2. People come up and talk about InnerSource
3. For corporate orgs, the main point is usually "lead gen" - in our case, an equivalent might be getting people onto Slack for example. 
4. The community has a focal point / place to gather

However, there are a lot of (sometimes hidden) costs to doing a booth. In my experience (15+ years organizing dev events) - booth participation rarely is the optimal activity for ROI at events. Here are some things to consider...

**Show me the money**
- Sometimes we get the booth slot for free, but sometimes you have to pay. A small booth at Kubecon for example might come with some tickets but cost c.$7K
- Then we pay for the booth stand, printing, giveaways, stickers etc. - and shipping - it's starts to mount up. 
- Event Staff (even if we have volunteers, you would typically have to have at least one staff member to organize everything be there for shipping etc. - time and travel. 

**Time is precious**
Things folks sometimes don't consider is the _time_ cost to having a booth:
- Orchestrating the booth plan
- Organizing logistics of getting booth stands / material to venue (if you have never tried carrying a box of T-shirts, you won't know how heavy they actually are)
- Training/prepping booth "staff" / volunteers
- Time on the booth itself 
- Takedown and orchestration of ship back

**Anything else?**
In the case of ISC - I'm assuming that the ask would be for volunteers to man the booth. A question to ask is - how much time will the community be able to give to man an ISC booth, if their organization has paid for them to be there to attend the event. (Hint: when we have asked this question in the past, we have not received many volunteers). 

**So what could we do instead?**
I would argue that some concentrated networking (e.g. showing up at event talks, asking questions featuring InnerSource, speaking to presenters can give you as much "awareness" as a booth. Our discussions in the past on this led to the ideas that became the idea of an [Event Lead](https://github.com/InnerSourceCommons/InnerSourceMarketing/commit/72f759558b516b10f620077e4a249a69e2d17f46) - which gives some of the benefits with a fraction of the money/time/effort commitment.

I would also argue that many booths resort to giveaways to get "leads" or contacts precisely because people wouldn't give their details anyhow - and would we want folks who wouldn't be signing up anyhow?

Also - for the time, effort and $ you might put into a booth at one event - if we had spare funds, I would argue they might be better put into a fund to get speakers to third party events to speak on behalf of ISC, who wouldn't otherwise be able to get there. I believe that would have a much higher return on awareness and connections.

**Comment by claredillon** [comment]
I forgot to add how absolutely exhausting it is to staff an event booth - something to factor in when asking our community to volunteer their time to it!

**Comment by rrrutledge** [comment]
We have the role of [Gathering Lead](https://github.com/InnerSourceCommons/InnerSourceMarketing/issues/new?assignees=&labels=Gathering+Lead&projects=&template=isc-gathering-lead.md&title=%5BCONFERENCE_NAME%5D+%5BCONFERENCE_DATE%28S%29%5D+%28Gathering+Lead%29) now that tries to have a lot of the same benefits that we are reaching for with the booth idea.
